Formula/p/promptfoo.rb

```diff
@@ -24,11 +24,8 @@ class Promptfoo < Formula
   def install
     ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version < 1700)
 
-    # Allow newer better-sqlite
-    # Backport of https://github.com/promptfoo/promptfoo/commit/9c70bbf438f65e38d7a026d8c42d63272c6ef801
-    inreplace "package.json", '"better-sqlite3": "12.4.1"', '"better-sqlite3": "12.4.6"'
     system "npm", "install", *std_npm_args
-    bin.install_symlink libexec.glob("bin/*")
+    bin.install_symlink Dir["#{libexec}/bin/*"]
```

Comment:
why revert this? We typically prefer pathnames

Reply:
do we have any docs for it? I always use `bin.install_symlink Dir["#{libexec}/bin/*"]`